apps/judicial-system/web/src/components/FormProvider/case.graphql

```diff
@@ -30,6 +30,7 @@ query Case($input: CaseQueryInput!) {
       serviceRequirement
       verdictViewDate
       verdictAppealDeadline
+      verdictAppealDate
```

Comment:
Is this field not needed for limited access users, such as defenders and prisons?

Reply:
Not for prisons at least but there might be a chance we'd want to display this for defenders. Thanks for pointing this out. We're discussing this in Asana, I'll come back to you on this when we've decided something.

Reply:
We decided to show this to defenders in another ticket so we don't need to add this for the limited access users in this pr.